README.md

```diff
@@ -66,8 +66,8 @@ formula = "on_b_a & O(ontable_c)"
 domain_parser = DomainParser()
 problem_parser = ProblemParser()
 
-domain = domain_parser(Path("examples/pddl/domain.pddl").read_text(encoding="utf-8"))
-problem = problem_parser(Path("examples/pddl/p-0.pddl").read_text(encoding="utf-8"))
+domain = domain_parser(Path("ignore/examples/pddl/domain.pddl").read_text(encoding="utf-8"))
+problem = problem_parser(Path("ignore/examples/pddl/p-0.pddl").read_text(encoding="utf-8"))
```

Comment:
to be fixed

Reply:
What do you think about:

```python


domain_path = Path("tests/benchmarks/deterministic/BF/blocksworld_ppltl/domain.pddl")
problem_path = Path("tests/benchmarks/deterministic/BF/blocksworld_ppltl/p2.pddl")

formula = "on_b_a & O(ontable_c)"
domain_parser = DomainParser()
problem_parser = ProblemParser()

domain = domain_parser(domain_path.read_text(encoding="utf-8"))
problem = problem_parser(problem_path.read_text(encoding="utf-8"))
goal = parse_pltl(formula)
```

Reply:
I tried to fix this at https://github.com/whitemech/Plan4Past/commit/120b58201688801c7907c74bc95b82830a193f94.

To add the changes on top of this PR you can do:

```
git cherry-pick 120b58
```

(or if we want to update the commit that introduced the problem, we would have to do a rebase).

However, I could not test locally, but it looks like it does not work :scream:  (fast-downward is not able to find a plan).

Reply:
ok, the path looks good. Why is not working? 😱

Reply:
the formula was not correct anymore because the constants were changed. I restored the previous example, I put the files in `tests/examples/blocksworld` (see [6b62472](https://github.com/whitemech/Plan4Past/pull/264/commits/6b624726aeec61fbccf59a03a51810de3cad55f2))

Reply:
ok, thank you